middleware/setBananaI18n.js

```diff
@@ -59,20 +59,12 @@ module.exports = (req, res, next) => {
     messages: config.messages,
   });
 
-  const translatedLocales = [];
   let activeLocale;
 
   locales.forEach((l) => {
```

Comment:
rather than iterating through each locale you can use find so it stops when you find the active one:

`const activeLocale = locales.find((l)  => l.localeCode === locale)`